src/pages/lessons/[slug].tsx

```diff
@@ -168,6 +170,11 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
     if (media_url) {
       axios.get(media_url).then(({data}) => {
         setMedia(data)
+        const event: PlayerStateEvent = {
```

Comment:
instead of doing oddball stuff in a switch statement, we will jump to `loaded` here when we have media